app/Http/Controllers/IgnoreProjectController.php

```diff
@@ -8,8 +8,13 @@ class IgnoreProjectController extends Controller
 {
     public function __invoke(Project $project)
     {
-        $project->update(['ignored' => true]);
-
+				$project->update(['ignored' => true]);
+				
+				if (request()->wantsJson()) {
```

Comment:
I believe we will always be returning json so we can remove this conditional.

Reply:
Fixed with latest commit